apps/nextra/components/landing/codeSnippets.ts

```diff
@@ -0,0 +1,123 @@
+export const coinsCodeSnippet = `
+module 0xCAFE::BasicCoin {
+  // Only included in compilation for testing. Similar to #[cfg(testing)]
+  // in Rust. Imports the \`Signer\` module from the MoveStdlib package.
+  #[test_only]
+  use std::signer;
+
+  struct Coin has key {
+      value: u64,
+  }
+
+  public fun mint(account: signer, value: u64) {
+      move_to(&account, Coin { value })
+  }
+
+  // Declare a unit test. It takes a signer called \`account\` with an
+  // address value of \`0xC0FFEE\`.
+  #[test(account = @0xC0FFEE)]
+  fun test_mint_10(account: signer) acquires Coin {
+      let addr = signer::address_of(&account);
+      mint(account, 10);
+      // Make sure there is a \`Coin\` resource under \`addr\` with a value of \`10\`.
+      // We can access this resource and its value since we are in the
+      // same module that defined the \`Coin\` resource.
+      assert!(borrow_global<Coin>(addr).value == 10, 0);
+  }
+}
+`.trim();
+
+export const objectsCodeSnippet = `
```

Comment:
I really like how comprehensive this is, but I think we should try to keep the snippets approximately the same length as the Coins snippet (approx 500px height) so that they're consistent. I've added some suggestions where I would cut (mostly around removing comments)

Reply:
I trimmed all the fat from the code snippets per your suggestions.